node/src/BaseClient.ts

```diff
@@ -1271,15 +1279,16 @@ export class BaseClient {
      * console.log(result); // Output: 11
      * ```
      */
-    public async incr(key: string): Promise<number> {
-        return this.createWritePromise(createIncr(key));
+    public async incr(key: GlideString, decoder?: Decoder): Promise<number> {
```

Comment:
please remove the decoder option.  This only affects the return value, and since we are returning a number, there is no decoding needed here.